client/src/pages/PublicProfiles/useStyles.js

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  dashboard: { backgroundColor: '#FFFFFF' },
+  pageContent: {
+    margin: '4rem auto',
+  },
+  addNewListIcon: {
+    fontSize: '4rem',
+  },
+  cardsContainer: {
+    margin: '2rem auto',
+  },
+  shoppingListsContentArea: {
+    margin: '4rem 0',
+    [theme.breakpoints.down('sm')]: {
+      margin: '4rem 1rem',
+    },
+  },
+  // duplicate styling here
+  shoppingListButton: {
+    height: '100%',
+    textAlign: 'center',
+  },
+  shoppingListsTitle: {
+    fontWeight: 700,
+  },
+  shoppingListCard: {
+    width: '270px',
+    height: '380px',
+    borderRadius: '0.8rem',
+  },
```

Comment:
If we have duplicated styling, couldn't this be moved to a shared folder where both components that need to use this style can grab it from?